src/pages/Landing/NetworkStatus/index.tsx

```diff
@@ -114,7 +106,11 @@ export const NetworkStatus: React.FC<{ amountEth?: number }> = ({
               </Heading>
               <Text size="x-large" className="mt20">
                 <BoldGreen className="mr10" fontSize={24}>
-                  {validatorAmount}
+                  {totalValidators === 0 ? (
```

Comment:
This logic (`totalValidators === 0`) to determine if the query is loading might be problematic because we return 0 if there's an error with the API.

Reply:
Will take a look at this now